src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -1,40 +1,42 @@
 package com.kodilla.ecommercee.controller;
 
 import com.kodilla.ecommercee.dto.ProductDto;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.web.bind.annotation.*;
 import java.util.ArrayList;
 import java.util.List;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 @RestController
 @RequestMapping("/v1/product")
+@Slf4j
 public class ProductController {
 
     @GetMapping(value = "all")
     public List<ProductDto> getProducts() {
-        System.out.println("Getting list of products");
+        log.info("Getting list of products");
         return new ArrayList<>();
     }
 
     @GetMapping("{productId}")
     public ProductDto getProduct(@PathVariable Long productId) {
-        System.out.println("Getting a product by id");
+        log.info("Getting a product by id");
         return new ProductDto();
     }
 
     @PostMapping(consumes = APPLICATION_JSON_VALUE)
     public void createProduct(@RequestBody ProductDto productDto) {
-        System.out.println("Creating new product.");
+        log.info("Creating new product.");
```

Comment:
Np. "Creating new product: " + productDto.getName() + ", price " + productDto.getPrice() + ", in group " + productDto.getGroupId()